apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-settings/vo-settings-expiration/vo-settings-expiration.component.html

```diff
@@ -1,5 +1,10 @@
-<app-expiration-settings
-  *ngIf="expirationAttribute !== undefined"
-  (saveAttribute)="saveExpirationAttribute($event)"
-  [expirationAttribute]="expirationAttribute">
-</app-expiration-settings>
+<ng-template #spinner>
+  <perun-web-apps-loading-table></perun-web-apps-loading-table>
+</ng-template>
+<ng-container *ngIf="expirationAttribute !== undefined; else spinner">
+  <app-expiration-settings
+    *perunWebAppsLoader="loading; indicator: spinner"
+    (saveAttribute)="saveExpirationAttribute($event)"
+    [expirationAttribute]="expirationAttribute">
+  </app-expiration-settings>
+</ng-container>
```

Comment:
The `position-relative` class is needed for the correct displaying of the spinner (to cover just a defined HTML).
```suggestion
<div class="position-relative">
  <app-expiration-settings
    *perunWebAppsLoader="loading; indicator: spinner"
    (saveAttribute)="saveExpirationAttribute($event)"
    [expirationAttribute]="expirationAttribute">
  </app-expiration-settings>
</div>
```